scripts/assemble_visualization_webpage.py

```diff
@@ -116,6 +119,11 @@ def get_primers_data(full_path, consensus):
     primers_metainfo["nCoV-2019"] = "ARTIC bioinformatics primers for nanopore sequencing of nCoV2019"
 
     primers_map = {}
+    if not full_path:
+        print("No primers table provided, skipping.")
+        return primers_map
+    else:
+        print("Parsing primers")
```

Comment:
Similar to the VCF and GFF parsers, it would be nice to print the filename (`full_path`?) here.

Reply:
I was hesitating. Given that we only provide one explicit CSV file on the command line, whereas we provide a directory which is scanned for annotations. Listing the GFF files as we scan the directory makes sense to me, I was a bit unsure about the primer CSV. 
Well, I'll add it.